test/nightly/getRotatingValue.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+let locationCount: number = -1;
+const locationItem: string[] = ['Australia East', 'Australia Southeast', 'Brazil South', 'Canada Central', 'Central US', 'East Asia', 'East US', 'East US 2', 'France Central', 'Japan East', 'Japan West', 'Korea Central', 'Korea South', 'North Europe', 'South Central US', 'Southeast Asia', 'UK South', 'West Europe', 'West US', 'West US 2'];
```

Comment:
This still needs to be closer to what's in https://github.com/microsoft/vscode-azurefunctions/blob/main/test/nightly/getRotatingValue.ts

For example, call it `getRotationLocation` instead of `getLocation` and use the same logic with `getStartingIndex` instead of the `new Date()` stuff